src/RootPipeTester/RootPipeTest.m

```diff
@@ -121,6 +121,8 @@ - (BOOL)runTestWithAuthorization:(BOOL)useAuth fileAttributes:(NSDictionary **)f
 		printf("The tool indicates that writing the file \"%s\" failed.\n", testFileStr);
 	}
 	
+	usleep((useAuth ? 5000 : 2500)); //fixes false negatives on 10.9 --> https://github.com/sideeffect42/RootPipeTester/issues/1 ; TODO test 10.10
```

Comment:
Must be 2500 at least for nil auth test and at least 5000 for auth test.
But if system load is big, we might consider even waiting up to several second.